config/core/resources/broker.yaml

```diff
@@ -42,31 +59,11 @@ spec:
     - knative
     - eventing
   scope: Namespaced
-  subresources:
-    status: {}
   conversion:
     strategy: Webhook
-    webhookClientConfig:
-      service:
-        name: eventing-webhook
-        namespace: knative-eventing
-  additionalPrinterColumns:
-    - name: Ready
-      type: string
-      JSONPath: ".status.conditions[?(@.type==\"Ready\")].status"
-    - name: Reason
-      type: string
-      JSONPath: ".status.conditions[?(@.type==\"Ready\")].reason"
-    - name: URL
-      type: string
-      JSONPath: .status.address.url
-    - name: Age
-      type: date
-      JSONPath: .metadata.creationTimestamp
-  versions:
-  - name: v1alpha1
-    served: false
-    storage: false
-  - name: v1beta1
-    served: true
-    storage: true
+    webhook:
+      conversionReviewVersions: ["v1", "v1beta1"]
+      clientConfig:
+        service:
+          name: eventing-webhook
+          namespace: knative-eventing
```

Comment:
Looks like you can't drop the v1alpha1 for the brokers based on this from the failed upgrade tests:
for: "STDIN": CustomResourceDefinition.apiextensions.k8s.io "brokers.eventing.knative.dev" is invalid: status.storedVersions[0]: Invalid value: "v1alpha1": must appear in spec.versions
Error from server (Invalid): error when applying patch:
Or:
for: "STDIN": CustomResourceDefinition.apiextensions.k8s.io "pingsources.sources.knative.dev" is invalid: status.storedVersions[0]: Invalid value: "v1alpha1": must appear in spec.versions

Reply:
Yeah this requires the storage migration job to be run _before_ the upgrade

[I do this after the upgrade](https://github.com/knative/serving/blob/master/test/e2e-upgrade-tests.sh#L56-L64) in serving which is technically wrong (we've never dropped a version yet so we didn't catch it)